readthedocs/templates/builds/build_detail.html

```diff
@@ -92,6 +92,29 @@
       </span>
     </div>
 
+    <div class="build-ideas">
+    {% if not build.success and build.commands.count < 4 %}
+      <p>
+
+        {% url 'wipe_version' build.version.project.slug build.version.slug as wipe_url %}
+        {% blocktrans %}
+        Having trouble with your build environment?
+        Try <a href="{{ wipe_url }}">reseting it</a>.
```

Comment:
`wipe_url` should be in the blocktrans variable list

Reply:
This is how the docs say to do it. https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#blocktrans-template-tag